base/cvd/cuttlefish/host/libs/web/credential_source.cc

```diff
@@ -73,13 +75,15 @@ Result<std::unique_ptr<CredentialSource>> GetCredentialSourceLegacy(
   } else if (credential_source == "") {
     LOG(VERBOSE) << "Probing acloud credentials at " << oauth_filepath;
     if (FileExists(oauth_filepath)) {
-      std::ifstream stream(oauth_filepath);
-      auto attempt_load =
-          RefreshCredentialSource::FromOauth2ClientFile(http_client, stream);
+      std::string oauth_contents;
+      CF_EXPECT(
+          android::base::ReadFileToString(oauth_filepath, &oauth_contents));
```

Comment:
More of an inquiry than a comment.  Do you think we be using this android helper instead of `file.h:ReadFileContents` (which is used elsewhere in this file)?  That version has more verbose error messages, but the implementations are quite different so I am not sure about all of the tradeoffs between one or the other.  

Mostly I would vote for consistency or making the reason for the diverging choice obvious.  I'll try to dig a bit deeper once I finish reviewing the rest of the changes.